### DISCUSSION TITLE
有计划在UMI4中支持 ahooks v3 么？

### DISCUSSION THREAD

**Discussion by punkin6** [umijs/umi#10784] [General]
需要使用一些ahooks v3的 react hook，目前UMI4 支持 ahooks v2，有计划在UMI4中支持 ahooks v3 么？

**Comment by fz6m** [comment]
自己安装下 ahooks v3 就能用了。

**Reply by xXAvoraXx** [reply]
這個配置是如何完成的呢？ahooks v3
https://umijs.org/docs/max/request#%E6%9E%84%E5%BB%BA%E6%97%B6%E9%85%8D%E7%BD%AE

**Reply by fz6m** [reply]
如需使用 ahooks v3 ，可以关闭 umi 的 request 功能，单独安装 ahooks 来使用。
